src/lib/features/personal-dashboard/personal-dashboard-controller.e2e.test.ts

```diff
@@ -61,3 +62,84 @@ test('should return personal dashboard with own flags and favorited flags', asyn
         ],
     });
 });
+
+const createProject = async (name: string, user: IUser) => {
+    const auditUser = {
+        id: 1,
+        username: 'audit user',
+        ip: '127.0.0.1',
+    };
+    const project = await app.services.projectService.createProject(
+        {
+            name,
+        },
+        user,
+        auditUser,
+    );
+    return project;
+};
+
+test('should return personal dashboard with membered projects', async () => {
+    // create project A with user 1
```

Comment:
those comments don't tell me anything I can't learn from the code itself

Reply:
fair; I'll remove them